lib/cli.js

```diff
@@ -39,6 +39,12 @@ const { argv } = yargs
     default: false,
     description: "Allow existing output files to be overwritten",
   })
+  .option("appendExt", {
+    type: "boolean",
+    default: false,
+    description:
+      "Append .avif to the file name instead of replacing the current extension",
```

Comment:
Could add the `(foo.jpg => foo.jpg.avif)` example here.

Reply:
Sure thing, now added 👍